public/javascripts/payments.js

```diff
@@ -573,6 +573,17 @@
     }
   };
 
+  /**
+   * Check if the PaymentIntent is in a "terminal" status
+   * and therefore if we should show an error in the UI
+   */
+  const paymentIntentTerminal = ({status, last_payment_error}) => {
```

Comment:
Maybe: `paymentIntentTerminalState`

So I got confused initially reading this code because I thought it was about Stripe Terminals! That's probably not something a regular user would do though.